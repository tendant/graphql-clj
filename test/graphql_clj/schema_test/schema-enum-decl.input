enum DogCommand {
  SIT
  DOWN
  HEEL
}
