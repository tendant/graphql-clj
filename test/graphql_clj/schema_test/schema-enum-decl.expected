{:enums {:DogCommand {:values [:SIT :DOWN :HEEL]}}}
